nuxeo-common/src/main/java/org/nuxeo/common/utils/StringUtils.java

```diff
@@ -192,4 +195,21 @@ public static String expandVars(String expression, Map<?, ?> properties) {
         return Vars.expand(expression, properties);
     }
 
+    /**
+     * Returns a json string representation of the object.
+     * <p>
+     * This helper does not check for circular reference and will lead to a stack overflow if any.
+     *
+     * @param object the object
+     * @return a json string or null if object is null
+     * @throws IOException
+     * @since 10.3
+     */
+    public static String toJson(Object object) throws IOException {
+        if (object == null) {
+            return null;
+        }
+        return new ObjectMapper().writeValueAsString(object);
```

Comment:
Creating a new `ObjectMapper` every time is very costly.

Reply:
:+1: